dmn-javascript-evaluation-provider/pom.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>dmn</artifactId>
+        <groupId>org.powerflows</groupId>
+        <version>2.0.0-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>dmn-javascript-evaluation-provider</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.powerflows</groupId>
+            <artifactId>dmn-model</artifactId>
+            <version>${parent.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.powerflows</groupId>
+            <artifactId>dmn-evaluation-provider-api</artifactId>
+            <version>${parent.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.powerflows</groupId>
+            <artifactId>dmn-io-api</artifactId>
+            <version>${parent.version}</version>
```

Comment:
evaluation provider requires IO?
tests may need it to load resources but otherwise it shouldn't be needed here

Reply:
No, it doesn't. Removed.